seshdash/utils/send_mail.py

```diff
@@ -3,11 +3,12 @@
 from django.template import Context
 import traceback
 
-def send_mail(subject,to_email):
+def send_mail(subject,to_email,content):
     try:
         plaintext = get_template('mail/site_mail.txt')
         htmly     = get_template('mail/site_mail.html')
-        d = Context({ 'username' : "User2"
+        d = Context({ 'username' : "User2",
+                      'content'  : content,
                       })
         from_email = "seshdash@gmail.com"
```

Comment:
Maybe we can add it to the local_settings.ini file then pull it in as a Constntin the settings.
Once it's a constant in the settings you can access it from the function.  I used to do something similar with the enphase api key.